build/tfs/common/symbols.ts

```diff
@@ -171,12 +171,12 @@ async function ensureVersionAndSymbols(options: IOptions) {
 }
 
 // Environment
-const pakage = require('../../../package.json');
+const pkg = require('../../../package.json');
```

Comment:
I understand that `package` is a reserved word.

Generally, instead of a random misspelling, people use an abbreviation, I've picked `pkg` (and Travis doesn't seem to object). I'm flexible, in case you have a preference. Other alternatives include using a `_`...